frontend/src/metabase/components/DateMonthYearWidget/DateMonthYearWidget.unit.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+
+import DateMonthYearWidget from "./DateMonthYearWidget";
+
+describe("DateMonthYearWidget", () => {
+  it("should render correctly", () => {
+    const { container } = render(
+      <DateMonthYearWidget
+        value={"2021-07"}
+        setValue={jest.fn()}
+        onClose={jest.fn()}
+      ></DateMonthYearWidget>,
+    );
+
+    expect(screen.getByText("January")).toBeDefined();
+    expect(screen.getByText("December")).toBeDefined();
```

Comment:
Please use the [`toBeInTheDocument`](https://github.com/testing-library/jest-dom#tobeinthedocument) or [`toBeVisible`](https://github.com/testing-library/jest-dom#tobevisible) assertions for UI

Reply:
Good idea, I've changed it accordingly.